modules/test/playwright/tests/layout-set-prototype-web/siteTemplatesWithPage.spec.ts

```diff
@@ -0,0 +1,149 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {applicationsMenuPageTest} from '../../fixtures/applicationsMenuPageTest';
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {pageEditorPagesTest} from '../../fixtures/pageEditorPagesTest';
+import {pagesAdminPagesTest} from '../../fixtures/pagesAdminPagesTest';
+import {productMenuPageTest} from '../../fixtures/productMenuPageTest';
+import {sitesPageTest} from '../../fixtures/sitesPageTest';
+import {uiElementsPageTest} from '../../fixtures/uiElementsTest';
+import getRandomString from '../../utils/getRandomString';
+import createSiteTemplate from './utils/createSiteTemplate';
+
+export const test = mergeTests(
+	applicationsMenuPageTest,
+	dataApiHelpersTest,
+	loginTest(),
+	pageEditorPagesTest,
+	pagesAdminPagesTest,
+	productMenuPageTest,
+	sitesPageTest,
+	uiElementsPageTest
+);
+
+test(
+	'Can not edit Pages created with Site Templates if the checkbox of allowing edition is disabled',
+	{tag: ['@LPD-49053', '@LPS-131903', '@LPS-132256']},
+	async ({
+		apiHelpers,
+		applicationsMenuPage,
+		page,
+		pageEditorPage,
+		pagesAdminPage,
+		productMenuPage,
+		sitesPage,
+		uiElementsPage,
+	}) => {
+
+		// Create a site template with the checkbox of allowing edition (layoutsUpdateable) disabled
+
+		const siteTemplateName: string = 'SiteTemplate-' + getRandomString();
+
+		const layoutSetPrototype = await createSiteTemplate({
+			apiHelpers,
+			layoutsUpdateable: false,
+			page,
+			productMenuPage,
+			templateName: siteTemplateName,
+		});
+
+		apiHelpers.data.push({
+			id: layoutSetPrototype.layoutSetPrototypeId,
+			type: 'layoutSetPrototype',
+		});
+
+		// Create and publish a page in the site template
+
+		const pageName: string = 'Page-' + getRandomString();
+
+		await productMenuPage.goToPages();
+		await pagesAdminPage.newButton.click();
+		await pagesAdminPage.addPage({
+			name: pageName,
+		});
+
+		await uiElementsPage.publishButton.click();
+
+		// Create a site using the site template
+
+		await applicationsMenuPage.goToSites();
+
+		const siteName: string = 'Site-' + getRandomString();
+
+		const siteId = await sitesPage.createSiteFromTemplate({
+			siteName,
+			templateName: siteTemplateName,
+		});
+
+		apiHelpers.data.push({id: siteId, type: 'site'});
+
+		// Check the Edit button of the dropdown is not visible
+
+		await productMenuPage.goToPages();
+
+		await page.reload();
```

Comment:
Do we really need this?

Reply:
Locally, sometimes it works but sometimes it does not. It seems after clicking the button, for any reason, the page loads without showing the available Pages. I saw that pattern [here](https://github.com/liferay/liferay-portal/blob/485dd2a6ff134f3066f9cd907c7d31a6505ca51c/modules/test/playwright/tests/frontend-js-web/session.spec.ts#L86)

Reply:
Its fine then! :)

Reply:
I will do it in a more elegant way